exercises/poker/canonical-data.json

```diff
@@ -0,0 +1,317 @@
+{
+  "exercise": "poker",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "it should be able to pick the winning poker hand",
+      "cases": [
```

Comment:
I'm not convinced a second level of `cases` nesting is necessary. I suggest just a single level, please, unless there is a specific reason to use two, and it doesn't seem that the extra description here adds any benefit. For example, https://github.com/exercism/x-common/blob/master/exercises/hello-world/canonical-data.json uses just one level.